File: GRLTest/Mixedrelationship1/Mixedrelationship1.c
#include <stdio.h>

double Mixedrelationship1( double E,double D,double C,double B,double A){
	double expr = 1.0*fmin(E, fmax(0, fmin(100.0, 0.75*D + 1.0*fmax(B, C))));
	return expr;
}

File: GRLTest/Mixedrelationship1/Mixedrelationship1.java
public class Model{
	public double Mixedrelationship1(double E,double D,double C,double B,double A){
		double expr = 1.0*Math.min(E, Math.max(0, Math.min(100.0, 0.75*D + 1.0*Math.max(B, C))));
		return expr;
	}
}
